pkgs/development/python-modules/bitsandbytes/default.nix

```diff
@@ -1,39 +1,44 @@
 {
   lib,
+  torch,
+  symlinkJoin,
   buildPythonPackage,
   fetchFromGitHub,
-  python,
-  pythonOlder,
+  cmake,
   setuptools,
-  wheel,
-  torch,
   scipy,
-  symlinkJoin,
 }:
 
 let
   pname = "bitsandbytes";
-  version = "0.43.1";
+  version = "0.44.1";
 
-  inherit (torch) cudaCapabilities cudaPackages cudaSupport;
-  inherit (cudaPackages) backendStdenv cudaVersion;
+  inherit (torch) cudaPackages cudaSupport;
+  inherit (cudaPackages) cudaVersion;
+
+  cudaVersionString = lib.replaceStrings [ "." ] [ "" ] (lib.versions.majorMinor cudaVersion);
 
   # NOTE: torchvision doesn't use cudnn; torch does!
   #   For this reason it is not included.
   cuda-common-redist = with cudaPackages; [
-    cuda_cccl # <thrust/*>
-    libcublas # cublas_v2.h
+    (lib.getDev cuda_cccl) # <thrust/*>
+    (lib.getDev libcublas) # cublas_v2.h
+    (lib.getLib libcublas)
     libcurand
     libcusolver # cusolverDn.h
-    libcusparse # cusparse.h
+    (lib.getDev libcusparse) # cusparse.h
+    (lib.getLib libcusparse)
+    libcusparse.lib
```

Comment:
drop the last? Btw `getDev libcusparse` has `.lib` in `propagatedBuildInputs`, I forget if that matters to `symlinkJoin`

Reply:
Oh, well the last line `libcusparse.lib` is clearly a mistake and is not needing because I have added `lib.getLib libcusparse` above.
Now, having said that, you suggest also getting rid of that and only keeping `lib.getDev libcusparse` right ?

Reply:
It works with `lib.getDev libcusparse` alone.